packages/mobile/src/screens/track-screen/TrackScreenDetailsTile.tsx

```diff
@@ -326,7 +326,7 @@ export const TrackScreenDetailsTile = ({
     ({ isPreview = false } = {}) => {
       if (isLineupLoading) return
 
-      if (isPlaying && isPreviewing === isPreview) {
```

Comment:
this is suspect but lgtm... why was it this way before

Reply:
Yeah let's follow up on this cause I didn't totally understand the rationale with this line